### PR TITLE
Variable for vertical orientation in Block class

### PR DESCRIPTION
**Pull request by astersnake**
## Proposed Changes

  - `vertical` variable and function add to block class 
  - if `vertical` is true in `block.blade.php` layout orientation will be vertical instead of horizontal
  
The reason behind this is to have the ability to use full horizontal space for layouts that have more information (tables for example) without missing the functionality of the Block class.

### CODE REVIEW

File: src/Screen/Layouts/Block.php [hunk 1]
```diff
@@ -37,6 +37,7 @@ abstract class Block extends Layout
     public function __construct(array $layouts = [])
     {
         $this->layouts = $layouts;
+        $this->variables['vertical'] = false;
```
Comment (tabuna) [inline src/Screen/Layouts/Block.php:40]: Let's use the property:

```php
protected $layouts = [
    'vertical' => false
];
```

File: resources/views/layouts/block.blade.php [hunk 2]
```diff
@@ -8,7 +8,7 @@
             </p>
         </legend>
     </div>
-    <div class="col-12 col-md-7 shadow-sm">
+    <div class="col-12 {{$vertical ? 'col-md-12' : 'col-md-7'}} shadow-sm">
```
Comment (tabuna) [inline resources/views/layouts/block.blade.php:11]: By default, full-width markup is used. We don't need to re-install it for a specific screen.

File: src/Screen/Layouts/Block.php [hunk 3]
```diff
@@ -86,6 +87,20 @@ public function description($description): self
...
     }
+    
+    /**
+     * Used to define block orientation.
+     *
+     * @param bool $vertical
+     *
+     * @return Block
+     */
+    public function vertical($vertical): self
```
Comment (tabuna) [inline src/Screen/Layouts/Block.php:98]: Let's make it possible to call this method without constantly passing the value `true`

```php
public function vertical($vertical = true)
```

Comment (astersnake) [reply src/Screen/Layouts/Block.php:98]: Ok, first I thought this but codeclimate scared me lol

File: src/Screen/Layouts/Block.php (hunk 1 above)
Comment (astersnake) [reply src/Screen/Layouts/Block.php:40]: I tried this but I got `Undefined variable`

Comment (astersnake) [reply src/Screen/Layouts/Block.php:40]: Nevermind, the variable name was `$variables`

File: resources/views/layouts/block.blade.php (hunk 2 above)
Comment (astersnake) [reply resources/views/layouts/block.blade.php:11]: So this should be updated to 

`` <div class="col-12 {{!$vertical ? 'col-md-7' : ''}} shadow-sm">``

Right?

**Comment by astersnake** [comment]
@tabuna I updated the pull request according to your suggestions

**Pull request merged**